### PR TITLE
Fix js client

### PR DESCRIPTION
**Pull request by kulmann**
The API endpoint for `BundleService.ListBundles` doesn't accept the `accountUuid` as param anymore. That was breaking the UI (fetching bundles failed, so the UI could not finish loading).

### CODE REVIEW

File: ui/client/settings/index.js [hunk 1]
```diff
@@ -395,6 +395,52 @@ export const BundleService_RemoveSettingFromBundleURL = function(parameters = {}
...
+/**
+ * 
+ * request: PermissionService_ListPermissionsByResource
+ * url: PermissionService_ListPermissionsByResourceURL
+ * method: PermissionService_ListPermissionsByResource_TYPE
+ * raw_url: PermissionService_ListPermissionsByResource_RAW_URL
+ * @param body - 
+ */
+export const PermissionService_ListPermissionsByResource = function(parameters = {}) {
+  const domain = parameters.$domain ? parameters.$domain : getDomain()
```
Comment (C0rby) [inline ui/client/settings/index.js:407]: Can this be replaced by `parameters.$domain || getDomain()`?

File: ui/client/settings/index.js [hunk 2]
```diff
@@ -395,6 +395,52 @@ export const BundleService_RemoveSettingFromBundleURL = function(parameters = {}
...
+  const config = parameters.$config
+  let path = '/api/v0/settings/permissions-list-by-resource'
+  let body
+  let queryParameters = {}
+  let form = {}
+  if (parameters['body'] !== undefined) {
```
Comment (C0rby) [inline ui/client/settings/index.js:413]: You could remove this if condition, if you move `let body = parameters['body']` after line 418.

File: ui/client/settings/index.js [hunk 3]
```diff
@@ -395,6 +395,52 @@ export const BundleService_RemoveSettingFromBundleURL = function(parameters = {}
...
+}
+export const PermissionService_ListPermissionsByResource_RAW_URL = function() {
+  return '/api/v0/settings/permissions-list-by-resource'
+}
+export const PermissionService_ListPermissionsByResource_TYPE = function() {
+  return 'post'
+}
+export const PermissionService_ListPermissionsByResourceURL = function(parameters = {}) {
+  let queryParameters = {}
+  const domain = parameters.$domain ? parameters.$domain : getDomain()
```
Comment (C0rby) [inline ui/client/settings/index.js:434]: Here also `parameters.$domain || getDomain()`

**Review by C0rby — APPROVED** [review]
Small remarks.
Otherwise looks good. :+1:

**Review by IljaN — APPROVED** [review]

File: ui/client/settings/index.js (hunk 1 above)
Comment (kulmann) [reply ui/client/settings/index.js:407]: it's a generated client ;-)

File: ui/client/settings/index.js (hunk 3 above)
Comment (kulmann) [reply ui/client/settings/index.js:434]: same as above, it's a generated client, so nothing we can do about it...

File: ui/client/settings/index.js (hunk 2 above)
Comment (kulmann) [reply ui/client/settings/index.js:413]: code generation ftw.. but yes, this could be improved 🤷

**Pull request merged**